include/fi_util.h

```diff
@@ -87,6 +87,9 @@
 	OFI_Q_READERR(prov, log, eq, "eq", fi_eq_readerr, 	\
 			fi_eq_strerror, ret, err_entry)
 
+#define ofi_sin_addr(addr) (((struct sockaddr_in *)(addr))->sin_addr)
+#define ofi_sin6_addr(addr) (((struct sockaddr_in6 *)(addr))->sin6_addr)
```

Comment:
Okay - I completely missed where those were used.  If you want to include this patch in the other PR, that's fine too